Cargo.toml

```diff
@@ -38,29 +38,29 @@ version = "20.3.4"
 path = "./cmd/crates/soroban-spec-tools"
 
 [workspace.dependencies.soroban-sdk]
-version = "=20.5.0"
+version = "=21.0.1-preview.1"
 
 [workspace.dependencies.soroban-token-sdk]
-version = "=20.5.0"
+version = "=21.0.1-preview.1"
 
 [workspace.dependencies.soroban-ledger-snapshot]
-version = "=20.5.0"
+version = "=21.0.1-preview.1"
 
 [workspace.dependencies.soroban-cli]
 version = "=20.3.4"
 path = "cmd/soroban-cli"
 
 [workspace.dependencies.soroban-rpc]
 package = "stellar-rpc-client"
-version = "=20.3.5"
+version = "=21.0.1"
 
 [workspace.dependencies.stellar-xdr]
-version = "=20.1.0"
+version = "=21.0.1"
 default-features = true
 
 [workspace.dependencies]
 stellar-strkey = "0.0.8"
-sep5 = "0.0.2"
+sep5 = "0.0.3"
```

Comment:
Reviewed this version bump and seems like a no-op as the only difference between sep5 0.0.2 and 0.0.3 is the change in dependence on stellar-strkey 0.0.7 to 0.0.8.

Reply:
Yeah it was causing issues because the version differences led to different types, but I had fixed this by going back to the raw type instead of returning the strkey type directly.